projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_configurator/product-configurator-vc-cart.e2e-flaky.cy.ts

```diff
@@ -84,7 +84,7 @@ context('Product Configuration', () => {
     configUISettings.productConfigurator.enableNavigationToConflict = false;
   });
 
-  describe('Navigate to Product Configuration Page', () => {
+  describe('Navigate to product configuration page', () => {
```

Comment:
Better do these changes in a separate PR (that would be of type test: rather than fix: )